factory/cryptoComponents.go

```diff
@@ -87,9 +92,14 @@ func (ccf *cryptoComponentsFactory) Create() (*CryptoComponents, error) {
 
 	txSignKeyGen := signing.NewKeyGenerator(ed25519.NewEd25519())
 
-	messageSignVerifier, err := systemVM.NewMessageSigVerifier(ccf.keyGen, singleSigner)
-	if err != nil {
-		return nil, err
+	var messageSignVerifier vm.MessageSignVerifier
```

Comment:
You can replace this line with: messageSignVerifier := &genesis.NilMessageSignVerifier{} and drop the else branch

Reply:
can't do it. as it says type missmatch